src/rulesets/Base/fastmath_able.jl

```diff
@@ -236,14 +236,48 @@ let
         end
 
         function rrule(::typeof(*), x::Number, y::Number)
-            project_x = ProjectTo(x)
-            project_y = ProjectTo(y)
-            function times_pullback(Ω̇)
+            function times_pullback2(Ω̇)
+                ΔΩ = unthunk(Ω̇)
+                return (NoTangent(), ProjectTo(x)(ΔΩ * y'), ProjectTo(y)(x' * ΔΩ))
+            end
+            return x * y, times_pullback2
+        end
+        function rrule(::typeof(*), x::Number, y::Number, z::Number)
+            function times_pullback3(Ω̇)
+                ΔΩ = unthunk(Ω̇)
+                return (
+                    NoTangent(), 
+                    ProjectTo(x)(ΔΩ * y' * z'),
+                    ProjectTo(y)(x' * ΔΩ * z'),
+                    ProjectTo(z)(x' * y' * ΔΩ),
+                )
+            end
+            return x * y * z, times_pullback3
+        end
+        # function rrule(::typeof(*), xyz::Number...)  # this version doesn't infer well
+        #     valN = Val(length(xyz))
+        #     function times_pullback4(Ω̇)
+        #         ΔΩ = unthunk(Ω̇)
+        #         Δxyz = ntuple(valN) do i
+        #             dxi = *(ntuple(j -> i==j ? ΔΩ : xyz[j]', valN)...)
+        #             ProjectTo(xyz[i])(dxi)
+        #         end
+        #         return (NoTangent(), Δxyz...)
+        #     end
+        #     return *(xyz...), times_pullback4
+        # end
```

Comment:
Do we want to leave this in the source?
I have mixed feeling about leaving dead commented out code.
Pro: easy to find, gives context.
Con: it is dead, it will easily become broken and outdated

Reply:
No, I think we should delete it.

Reply:
Maybe with a comment to say we tried the obvious n-arg thing, and it didn't infer well, with a link here?